src/libaudqt/prefs-window-qt.cc

```diff
@@ -1,6 +1,6 @@
 /*
  * prefs-window.cc
- * Copyright 2006-2014 William Pitcock, Tomasz Moń, Michael Färber, and
+ * Copyright 2006-2014 William Pitcock, Tomasz Mo?, Michael F�rber, and
```

Comment:
Might want to check that your editor is set to handle UTF-8 correctly.  Besides that, there don't seem to be any code changes left in this file, so please revert it to the version in master.